korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview1-1008
-commithash:e82dffbe8cf67ad9c07b4385cb6a1a1fc8be9fb1
+version:2.1.0-preview1-1009
```

Comment:
Waiting on http://aspnetci/viewLog.html?buildId=396852&tab=buildResultsDiv&buildTypeId=Lite_Public_DnxTools to complete so we have a build tools with the right stuff in it